src/components/collection/SelectProject.js

```diff
@@ -23,21 +23,23 @@ const selectStyle = {
         ...base,
         zIndex: "999"
     }),
-
 };
 
+/**
+ * React component for selecting list of projects from collection
+ */
 class SelectProject extends PureComponent {
+
+    /**
+     * Fetch the selected scene when the project is selected from the collection
+     * @param {Event} projectID ID of the project to be fetch
+     */
     handleChange = (projectID) => {
         // show warning if there's unsaved change
         if(this.hasEditorChanged()){
-            if((window.confirm("Are you sure you want to continue?\nYou will lose any unsaved work!"))){
-                if (this.props.user && this.props.user.uid) {
```

Comment:
So this code basically does nothing?  I'm guessing we're doing this since we're directing off to another page, which would refresh the editor anyways

Reply:
That part of the code is exactly same as the code below (when the code doesn't have any changes), it's redundant. So I flip the if statement to return if user doesn't want to proceed.